packages/react-adminer/src/components/TableFilter.tsx

```diff
@@ -99,8 +99,14 @@ const TableFilter: React.FC<Props> = ({ setWhere, setPage, config, where, entity
 					<Row gutter={16}>
 						{onlyStringConfigs.map(c => (
 							<Col key={c.name} span={12}>
-								{/* <Form.Item name={c.name} label={capitalCase(c.name)}> */}
-								<Form.Item name={c.name} label={c.name}>
+								<h2>
+									<Form.Item
+										name={c.name}
+										label={getColumnTitle(config.fields.filter(f => f.name === c.name)[0])}
```

Comment:
1. Tohle je ukazkovy priklad utilitni funkce, takze do utils
2. k tomu se pouziva find, ne filter
3. co kdyz na indexu nula nic nebude? tak to crashne cely a nebudes vedet proc, v tomhle pripade je dobry (kdyz nenajdes neco) udelat throw error(config not found) nebo tak neco, tim zabijes cely script, ale budes alespon neco vedet.